<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Personal Information:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditcusname">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonnewselling">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>New Selling</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtoncalculate">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>100</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditquantity">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantity:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditpayment">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonexit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Payment:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer is Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonstatistic">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>150</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistic</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Statistic Information</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total number of Customers</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdittotalnumber">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total student custon</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdittotalstudent">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdittotalrev">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Revenue</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonexit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
